packages/Bio/src/tests/activity-cliffs-utils.ts

```diff
@@ -1,11 +1,13 @@
 import * as DG from 'datagrok-api/dg';
 import * as grok from 'datagrok-api/grok';
 
-import {delay, expect} from '@datagrok-libraries/utils/src/test';
+import {expect} from '@datagrok-libraries/utils/src/test';
 import {activityCliffs} from '../package';
-import { DimReductionMethods } from '@datagrok-libraries/ml/src/reduce-dimensionality';
+import {DimReductionMethods} from '@datagrok-libraries/ml/src/reduce-dimensionality';
 
-export async function _testActivityCliffsOpen(df: DG.DataFrame, numberCliffs: number, method: DimReductionMethods, colName: string) {
+export async function _testActivityCliffsOpen(
+  df: DG.DataFrame, numberCliffs: number, method: DimReductionMethods, colName: string
+) {
```

Comment:
Please, do not add unnecessary lines. This could have been written in two lines: put as many parameters as you can on the line with the function name and the rest on the next line, including the closing bracket.
Also if the line ends with a parameter, it should have a comma in the end, no matter if it's the last parameter.

Reply:
> Also if the line ends with a parameter, it should have a comma in the end, no matter if it's the last parameter.

I don't think we should require a comma after the last parameter, why would we?

Reply:
there is actually a good reason to this. for example, It is particular useful when adding, removing, or reordering items in a list that spans multiple lines, because it reduces the number of lines that need to be changed, which helps with both editing and reviewing the diff. you can also refer to this [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas)

Reply:
I am against forcing developers to use trailing commas "just in case". Adding a new one is easy, and the diff looks just fine. You link just says that it's "particularly useful when reodering" - well, sort of, but how often do you reorder? 

There are places when it's useful and I'm not against it. But this is totally not an issue and should not be a rule.